conversions/decimal_to_hexadecimal.py

```diff
@@ -22,44 +22,34 @@
 
 def decimal_to_hexadecimal(decimal):
     """
-        take integer decimal value, return hexadecimal representation as str 
+        take integer decimal value, return hexadecimal representation as str beginning with 0x
         >>> decimal_to_hexadecimal(5)
-        '5'
+        '0x5'
         >>> decimal_to_hexadecimal(15)
-        'f'
+        '0xf'
         >>> decimal_to_hexadecimal(37)
-        '25'
+        '0x25'
         >>> decimal_to_hexadecimal(255)
-        'ff'
+        '0xff'
         >>> decimal_to_hexadecimal(4096)
-        '1000'
+        '0x1000'
         >>> decimal_to_hexadecimal(999098)
-        'f3eba'
+        '0xf3eba'
         >>> # negatives work too
         >>> decimal_to_hexadecimal(-256)
-        '-100'
+        '0x-100'
```

Comment:
Please add a test:
```
>>> decimal_to_hexadecimal(-256) == hex(-256)
True
```

Reply:
Done.  This also made me realize that I was supposed to put '-' before '0x', which I fixed